config/locales/en.yml

```diff
@@ -0,0 +1,16 @@
+en:
+  mountain_view:
+    main:
```

Comment:
Wondering on `main` vs `layout` for this key. Thoughts?

Reply:
I can go either way, I'd picked "main", but had also considered "defaults".  My thought was these would be used in many places rather than in just the layout, or a specific views.